### ISSUE TITLE
Add "Announcements" and notify them and "Notes"

### ISSUE THREAD

**Issue by zjhxmjl** [koweb/Vermis#11]
hey @omarpiani 
    for normal users:  
i think "Notes " should be random display when i refresh the page. Where is there a good place to put Notes?like my upload picture which include the color(maybe any places are better).
    for admin:
i think admin can edit the Notes when login in background.
![2017-03-04-6_scrot](https://cloud.githubusercontent.com/assets/4043922/23576215/fc074d32-00d9-11e7-9d2a-6b2785fab8a4.png)

**Comment by omarpiani** [comment]
❓ ❔

You want that, one random note, of a random project to be shown in the green area of your uploaded screenshot (that is dashboard). Really?

 😕

**Comment by zjhxmjl** [comment]
i only want random note in the green area,maybe i think it's name 
"announcement" .because i see the plugin for MantisBT 
https://github.com/mantisbt-plugins/announce,  so i get the idea.

**Comment by omarpiani** [comment]
when you mention "note" you are not talking about "Vermis" notes.

You want "admin" announcements, not related to projects. But... why random?

**Comment by zjhxmjl** [comment]
just like the announcements plugin for MantisBT

lets privileged accounts create and post announcements that can be shown 
to users on a global or per-project basis, and allow users to dismiss 
individual messages.

**Comment by omarpiani** [comment]
How about to popup announcements with something like that?

https://notifyjs.com/

2017-03-06 13:24 GMT+01:00 zjhxmjl <notifications@github.com>:

**Comment by zjhxmjl** [comment]
it's ok.it use javascript?

**Comment by omarpiani** [comment]
yes. jQuery. Obviously you need to get data from server side too. Is a quite big work.

**Comment by zjhxmjl** [comment]
ok,i see.maybe it should be able to be made into a plugin.
